CHANGES.md

```diff
@@ -2,6 +2,436 @@
 Release Changes List
 ====================
 
+TODO: search for docs/master and docs/1.13 and docs/latest
+TODO: modules/internal -> buildins?
+TODO: reorder sections
+
+*** stopped at 0290672, 2016-09-25 ***
+
+==============
+version 1.14.0
+==============
+
+Seventeenth public release of Chapel, October 6, 2016
+
+Highlights
+----------
+* added a new standard 'BigInteger' module providing value-based 'bigint's
+* new package modules for MPI, BLAS, ZeroMQ, MatrixMarket, and range chunking
+* dramatically revamped the Sort and Search modules, including comparators
+* added support for Block-distributed sparse domains and arrays
+* added a new 'StencilDist' distribution in support of stencil access patterns
+* added support for standard math routines on 'complex' values
+* added a capability for generating stack traces on 'halt's
+* added the ability to implement Chapel's atomics using C atomics
+* optimizations for array accesses, memory management, reductions, promotion
+* locality-oriented optimizations for block transfers, references, task counters
+* implemented 'sync' variables natively when using 'qthreads' tasking
+* improved support for reduce intents on parallel loops
+* added the ability to pass Chapel functions to extern C routines
+* added support for a Chapel 'docker' image
+* added the primer example codes to the online documentation
+* completed and updated our suite of Computer Language Benchmark Game codes
+* added several new pages to the online users guide
+* made significant improvements to the 'chplvis' execution analysis tool
+* added support for a new .chplconfig file supporting default CHPL_* settings
+* clarified errors related to modifying 'const' intent-based shadow variables
+* third-party package updates: jemalloc, GASNet, qthreads, hwloc, massivethreads
+* closed several classes of memory leaks, most notably due to sync/single vars
+* improved the portability of the code base
+* numerous bug fixes
+
+Packaging
+---------
+* added support for a Chapel 'docker' image
+  (see https://hub.docker.com/r/chapel/chapel/)
+  TODO: Should we pull this?  Does it relate to the release?  How soon
```

Comment:
I think we agreed to drop this, given that we never mentioned other packaging endeavors in changes files (e.g. homebrew)